misc/verbs/atrium-verbs-common/src/main/kotlin/ch/tutteli/atrium/api.verbs/AssertionVerb.kt

```diff
@@ -7,7 +7,7 @@ import ch.tutteli.atrium.reporting.translating.Translatable
  * The [Translatable]s for the assertion functions [assert], [assertThat] and [expect].
  */
 enum class AssertionVerb(override val value: String) : StringBasedTranslatable {
-    ASSERT("assert"),
-    ASSERT_THAT("assert that"),
-    EXPECT("expect"),
+    ASSERT("asserted that subject"),
+    ASSERT_THAT("asserted that subject"),
+    EXPECT("expected that subject"),
```

Comment:
There are things I didn't modify. Is it changes coming from the master?

Reply:
Yes, those are changes from master which you have re-applied on your branch and are thus seen as changes. You don't get this if you rebase on master (for a guideline for your next PR see https://github.com/robstoll/atrium/blob/master/.github/CONTRIBUTING.md#git). You can do the following to get rid of the commits not belonging to you (please commit all your local changes first):
- `git checkout -b t` to have a backup branch pointing at your current state in case you mess something up. This way you can always to a `git reset --hard t`. 
- `git checkout 103-get-ispresent`  back to your branch
- `git reset --soft 49638d85` kind of un-commit all changes (expect the first) to the point where you started with your first commit
- `git commit --amend` add all changes to your first commit
- `git remote add upstream https://github.com/robstoll/atrium.git`
- `git fetch upstream`
- `git rebase upstream/master` put your changes on top of Atrium's master
- `git push -f`  force push your commit (will update the PR)
- `git branch -D t` delete the tmp branch as you don't need it any more

I hope that helps

Reply:
Got it. I already configured the upstream. I think I didn't fetch from upstream though. Do I need to explicitly fetch from upstream every time or a regular fetch alredy? Thanks!